### PR TITLE
refactor(rxFeedbackTypes): rx-prefixes update feedbackTypes

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1168

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/elements/Feedback/scripts/rxFeedback.js [hunk 1]
```diff
@@ -142,15 +142,15 @@ angular.module('encore.ui.elements')
...
+.directive('rxFeedback', function (rxFeedbackTypes, $location, rxFeedbackSvc, rxScreenshotSvc, rxNotify, Session) {
     return {
         restrict: 'E',
         templateUrl: 'templates/rxFeedback.html',
         scope: {
             sendFeedback: '=?onSubmit'
         },
         link: function (scope) {
-            scope.feedbackTypes = feedbackTypes;
+            scope.rxFeedbackTypes = rxFeedbackTypes;
```
Comment (CITguy) [inline src/elements/Feedback/scripts/rxFeedback.js]: Please revert `scope.rxFeedbackTypes` back to `scope.feedbackTypes`.

File: src/elements/Feedback/templates/feedbackForm.html [hunk 2]
```diff
@@ -15,8 +15,8 @@
           <select rx-select
               id="selFeedbackType"
               ng-model="fields.type"
-              ng-options="opt as opt.label for opt in feedbackTypes"
-              ng-init="fields.type = feedbackTypes[0]"
+              ng-options="opt as opt.label for opt in rxFeedbackTypes"
```
Comment (CITguy) [inline src/elements/Feedback/templates/feedbackForm.html]: Please revert back to `feedbackTypes`.

File: src/elements/Feedback/templates/feedbackForm.html [hunk 3]
```diff
@@ -15,8 +15,8 @@
...
+              ng-init="fields.type = rxFeedbackTypes[0]"
```
Comment (CITguy) [inline src/elements/Feedback/templates/feedbackForm.html]: Please revert back to `feedbackTypes`.

File: src/elements/Feedback/scripts/rxFeedback.js (hunk 1 above)
Comment (cathysiller) [inline src/elements/Feedback/scripts/rxFeedback.js]: reverted

File: src/elements/Feedback/templates/feedbackForm.html (hunk 2 above)
Comment (cathysiller) [inline src/elements/Feedback/templates/feedbackForm.html]: reverted

File: src/elements/Feedback/templates/feedbackForm.html (hunk 3 above)
Comment (cathysiller) [inline src/elements/Feedback/templates/feedbackForm.html]: reverted

File: src/elements/Feedback/scripts/rxFeedback.js [hunk 4]
```diff
@@ -142,15 +142,15 @@ angular.module('encore.ui.elements')
...
+.directive('rxFeedback', function (rxFeedbackTypes, $location, rxFeedbackSvc, rxScreenshotSvc, rxNotify, Session) {
     return {
         restrict: 'E',
         templateUrl: 'templates/rxFeedback.html',
         scope: {
             sendFeedback: '=?onSubmit'
         },
         link: function (scope) {
-            scope.feedbackTypes = feedbackTypes;
+            scope.FeedbackTypes = rxFeedbackTypes;
```
Comment (CITguy) [inline src/elements/Feedback/scripts/rxFeedback.js]: Please use lowerCamelCase (`scope.feedbackTypes`)

File: src/elements/Feedback/templates/feedbackForm.html [hunk 5]
```diff
@@ -15,8 +15,8 @@
           <select rx-select
               id="selFeedbackType"
               ng-model="fields.type"
-              ng-options="opt as opt.label for opt in feedbackTypes"
-              ng-init="fields.type = feedbackTypes[0]"
+              ng-options="opt as opt.label for opt in FeedbackTypes"
```
Comment (CITguy) [inline src/elements/Feedback/templates/feedbackForm.html]: Please use lowerCamelCase (`feedbackTypes`).

File: src/elements/Feedback/templates/feedbackForm.html [hunk 6]
```diff
@@ -15,8 +15,8 @@
...
+              ng-init="fields.type = FeedbackTypes[0]"
```
Comment (CITguy) [inline src/elements/Feedback/templates/feedbackForm.html]: Please use lowerCamelCase (`feedbackTypes`).

File: src/elements/Feedback/scripts/rxFeedback.js (hunk 4 above)
Comment (cathysiller) [inline src/elements/Feedback/scripts/rxFeedback.js]: lower-cased "f" in "feedbackTypes"

File: src/elements/Feedback/templates/feedbackForm.html (hunk 5 above)
Comment (cathysiller) [inline src/elements/Feedback/templates/feedbackForm.html]: lower-cased "f" in "feedbackTypes"

File: src/elements/Feedback/templates/feedbackForm.html (hunk 6 above)
Comment (cathysiller) [inline src/elements/Feedback/templates/feedbackForm.html]: lower-cased "f" in "feedbackTypes"

**Comment by lentusignavus** [comment]
Should the folder be renamed to `rxFeedback`?

**Comment by CITguy** [comment]
@lentusignavus: utility object directories get named after the utility. Since we're dealing with `rxFeedbackTypes`, the directory `src/utilities/rxFeedbackTypes` is where its logic will reside.

**Comment by lentusignavus** [comment]
@CITguy gotcha

**Comment by lentusignavus** [comment]
DEV LGTM

**Pull request merged**